include/roaring/roaring.h

```diff
@@ -555,6 +555,20 @@ void roaring_bitmap_to_uint32_array(const roaring_bitmap_t *r, uint32_t *ans);
  */
 bool roaring_bitmap_to_bitset(const roaring_bitmap_t *r, bitset_t *bitset);
 
+/**
+ * Like `roaring_bitmap_to_bitset`, but store each bit in a single byte
+ * and output a array of `bool` in `ans` until `max_value`. For index `i` of
+ * `ans`, its value is true if `i` is set in roaring bitmap `r`, and false
+ * otherwise.
+ *
+ * Caller is responsible to ensure that there is enough memory allocated,
+ * e.g.
+ *     ans = malloc(roaring_bitmap_maximum(bitmap) * sizeof(bool));
+ *
+ * This function always returns `true`
```

Comment:
If the function always return true, why have a return value at all?

Reply:
> If the function always return true, why have a return value at all?

I followed the style of existing APIs like this one:

https://github.com/RoaringBitmap/CRoaring/blob/d5f843304265e41b5d4ae3104f636f28f858c9d8/include/roaring/roaring.h#L558-L573

Should we remove them all?